Source/bmalloc/libpas/src/libpas/pas_create_basic_heap_page_caches_with_reserved_memory.c

```diff
@@ -111,6 +134,21 @@ pas_basic_heap_page_caches* pas_create_basic_heap_page_caches_with_reserved_memo
         allocate_from_megapages,
         pas_megapage_cache_size_medium);
 
+    pas_megapage_cache_construct(
+        &caches->small_compact_exclusive_segregated_megapage_cache,
+        allocate_from_large,
+        pas_megapage_cache_size_small_compact);
+
+    pas_megapage_cache_construct(
+        &caches->small_compact_other_megapage_cache,
+        allocate_from_large,
+        pas_megapage_cache_size_small_compact);
```

Comment:
Do these strictly need to be separate for functional reasons, or is it just for clarity's sake? I'm OK with "for clarity's sake" I just want to understand if the code would break sans this.

Reply:
Resolved offline.